recipes/ftxui/all/test_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.verbose = True
```

Comment:
```suggestion
```

Reply:
Any reason not to do use that in test? It's very useful when inspecting the logs, and isn't too noisy.

Reply:
consistency across recipes.